ansible/roles/tasks-systemd/job-runner.toml

```diff
@@ -0,0 +1,25 @@
+[logs]
+driver='s3'
+
+[forge.github]
+token = [{file="/run/secrets/webhook/.config--github-token"}]
+
+[logs.s3]
+# bots lib/stores.py LOG_STORE
+url = 'https://cockpit-logs.us-east-1.linodeobjects.com/'
+key = [{file="/run/secrets/tasks/s3-keys/cockpit-logs.us-east-1.linodeobjects.com"}]
+
+[container]
+command = ['podman-remote', '--url=unix:///podman.sock']
+run-args = ['--device=/dev/kvm']
```

Comment:
That's going to need to get a bit bigger.  In addition to the issue about the cache dir you pointed out, at least these are missing:
 - `--memory=24g`
 - `--pids-limit=16384`
 - `--shm-size=1024m`
 - `${TMPVOL}` moral equivalent

Reply:
Done. Confirmed in podman exec, and I also did a podman run to get the full joy of 8x parallelism: https://cockpit-logs.us-east-1.linodeobjects.com/test-podman-1/log.html .